test/Interop/Cxx/value-witness-table/custom-virtual-destructors.swift

```diff
@@ -0,0 +1,56 @@
+// With RTTI some of the objects with virtual bases / destructors in this test
```

Comment:
Please rename the test to follow the pattern. Since it imports CustomDestructors, it should be named `custom-destructors-irgen.swift`.